frontend/src/components/placementVisuals/Placement.tsx

```diff
@@ -1,20 +1,26 @@
 import React, { useCallback, useEffect, useMemo } from "react"
 import { FMSId } from "../../models/fms_api_models"
 import { useAppDispatch, useAppSelector } from "../../hooks"
-import { Button, Col, Popconfirm, Radio, RadioChangeEvent, Row, Switch } from "antd"
+import { Button, Col, Popconfirm, Radio, RadioChangeEvent, Row, Switch, notification } from "antd"
```

Comment:
We forget that I wrote a notification system 😢 (`frontend/src/modules/notification/actions.tsx`)

Reply:
But eh I don't mind it not being used.

Reply:
Mmm oh i thought you were using antd. I'll look it up.